### PR TITLE
MSVC support

### PR DESCRIPTION
**Pull request by clin1234**

### CODE REVIEW

**Review by guerinoni — CHANGES_REQUESTED** [review]
Very good! Thank you! I want only suggest to change the position of `&` in order to be compliant with .clang-format

**Comment by guerinoni** [comment]
Rebase it, don't "reverse merge"

**Review by clin1234 — COMMENTED** [review]
Sure

File: src/app/main.cpp [hunk 1]
```diff
@@ -10,6 +10,7 @@ int main(int argc, char *argv[])
     QGuiApplication::setApplicationName("unofficial-discord-client");
     QGuiApplication::setOrganizationName("Coding Bunker");
     QGuiApplication::setApplicationVersion("0.0.1");
+    // QGuiApplication;
```
Comment (guerinoni) [inline src/app/main.cpp:13]: This?

**Review by guerinoni — CHANGES_REQUESTED** [review]

**Comment by guerinoni** [comment]
Rebase correctly in order to remove `e4d85732379fc6b5efead54879f20cc76ab4607e`

**Pull request closed without merging**